File: 0283-move-zeroes/0283-move-zeroes.c
void moveZeroes(int* nums, int numsSize) {
    int count = 0, k = 0;
    int new[numsSize];
    for(int i = 0 ; i < numsSize ; i++){
        if(nums[i] != 0){
            new[k] = nums[i];
            k++;
        }
    }
    for(int i = 0 ; i < k-numsSize ; i++){
        new[k] = 0;
    }
    return new[k];
}

File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        n = len(nums)
        k = 0

        for i in range(n):
            if nums[i] != 0:
                nums[k] = nums[i]
                k += 1

        while k < n:
            nums[k] = 0
            k += 1